partner_event/models/event_event.py

```diff
@@ -2,6 +2,7 @@
 # Copyright 2015 Tecnativa S.L. - Javier Iniesta
 # Copyright 2016 Tecnativa S.L. - Antonio Espinosa
 # Copyright 2016 Tecnativa S.L. - Vicent Cubells
+# Copyright 2018 Jupical Technologies Pvt. Ltd. - Anil Kesariya
```

Comment:
You shouldn't put your copyright on files that you haven't touched. Put your name on contributors part on the README, and copyright on the manifest file + touched files.